src/routes/+layout.svelte

```diff
@@ -19,7 +19,9 @@
 <svelte:head>
 	<title>{data.websiteTitle}</title>
 	<meta name="viewport" content={data.viewportWidth} />
-	<meta name="description" content={data.websiteShortDescription} />
```

Comment:
I think you can use `$page.data.websiteShortDescription` (maybe rename it to something else)

Then, it will default to the layout's `websiteShortDescription`, but a page could also return it in its data, and it will then take priority.